components/bt/esp_ble_mesh/models/server/time_scene_server.c

```diff
@@ -386,7 +387,8 @@ static void time_set(struct bt_mesh_model *model,
         change.time_set.subsecond = srv->state->time.subsecond;
         change.time_set.uncertainty = srv->state->time.uncertainty;
         change.time_set.time_authority = srv->state->time.time_authority;
-        change.time_set.tai_utc_delta_curr = srv->state->time.subsecond;
+        change.time_set.tai_utc_delta_curr = srv->state->time.tai_utc_delta_curr;
+        change.time_set.time_zone_offset_curr = srv->state->time.time_zone_offset_curr
```

Comment:
```suggestion
        change.time_set.time_zone_offset_curr = srv->state->time.time_zone_offset_curr;
```

Reply:
Sorry about that, split this out of a larger patch series and didn't notice there was a fixup in a later commit.

Reply:
@forx157 friendly nudge.  I pushed the semicolon.

Reply:
Hi, @s0be 
The changes you've committed have been merged into the internal branch and will be synced to github in the future!